style.css

```diff
@@ -381,4 +385,207 @@ fieldset textarea {
   align-self: center;
 }
 
-/* --- End: Form Section--- */
+/* --- End: Form Section --- */
+
+/*************     Media Queries     *************************/
+
+/** Start: Desktop Navbar **/
+
+@media only screen and (width >= 768px) {
+  .wrapper {
+    margin-bottom: 114px;
+    border-bottom-left-radius: 120px;
+  }
+
+  .container {
+    max-width: 100%;
+    width: 80%;
+    margin: 0 auto;
+  }
+
+  nav {
+    padding: 25px 0;
+    display: flex;
+  }
+
+  .fa-bars {
+    display: none;
+  }
+
+  nav ul {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    gap: 32px;
+  }
+
+  nav li {
+    list-style-type: none;
+  }
+
+  nav a {
+    text-decoration: none;
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    padding: 8px;
+  }
+
+  nav a:hover {
+    color: #6070ff;
+  }
+
+  /** End: Desktop header section **/
+
+  /** Start: Desktop Main section **/
+
+  .main {
+    background-image: url("images/main-bg-desktop.svg");
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+  }
+
+  .main-section {
+    margin: 0 auto;
+    width: 63%;
+    padding: 260px 0;
+    gap: 12px;
+    height: auto;
+  }
+
+  .main-section h1 {
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    width: max-content;
+    transition: transform 2s;
+  }
+
+  .main-section p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .main-section h1 span {
+    display: block;
+  }
+
+  /** End: Desktop Main section **/
+
+  /** Start: Desktop Work section **/
+
+  .work-history {
+    grid-row-gap: 142px;
+    padding: 24px;
+  }
+
+  .card {
+    flex-direction: row;
+  }
+
+  .row-reverse {
+    flex-direction: row-reverse;
+  }
+
+  .card img {
+    max-width: 100%;
+    width: 47%;
+  }
+
+  .card-inner {
+    max-width: 100%;
+    width: 47%;
+    padding: 4.4% 0 5.4% 4.4%;
+    gap: 24px;
+  }
+
+  .card-inner h2 {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+  }
+
+  .card-inner p {
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .card-inner p.detail {
+    font-size: 16px;
+  }
+
+  .description-container {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+  }
+
+  .tag-container {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+  }
+
+  /** End: Desktop Work section **/
+
+  /** Start: Desktop About section **/
+
+  .about {
+    display: flex;
+    flex-direction: row;
+    padding: 142px;
+    border-top-right-radius: 120px;
+    justify-content: flex-start;
+  }
+
+  .about-right > ul {
+    gap: 0;
+  }
+
+  .about-left {
+    flex-basis: 41%;
+  }
+
+  .about-right {
+    flex-basis: 59%;
+  }
+
```

Comment:
Nice work so far. However, kindly make sure that the left and right sides of the about section maintains have the same height just as in the figma design at all times, and at all screen sizes of the desktop version. The left side seems to be longer than the right side for screen widths below 950px as shown below :point_down: Kindly check and update accordingly. :pray: 
Kindly also consider adjusting the widths to 40% for left side and 60% to right side to match the design. 

### Your design 

![Screenshot from 2023-07-12 12-06-36](https://github.com/hamzaw9/portfolio/assets/65233860/66d689ba-0bb9-49d1-8d37-9b6591493ecd)



### Actual design 


![Screenshot from 2023-07-12 12-05-45](https://github.com/hamzaw9/portfolio/assets/65233860/dd6bd6a0-c54b-4465-b425-0d2396623208)

Reply:
Fixed.